plugins/omar-oms-plugin/grails-app/controllers/omar/oms/ChipperController.groovy

```diff
@@ -513,10 +512,10 @@ class ChipperController {
           a[b?.key] = b?.value?.toString()
           a
        }
-       println initOps
+       log.info initOps
```

Comment:
Don't think initOps is a String so need to convert it:

 log.info initOps as String